client/question/components/LearnerViewDirective.js

```diff
@@ -176,13 +182,35 @@ tie.directive('learnerView', [function() {
           border: 1px solid #e4e4e4;
         }
         .tie-button-blue {
-          background-color: rgb(66, 133, 244);
+          background-color: rgb(110, 150, 190);
           color: #ffffff;
         }
         .tie-button-blue:hover {
-          background-color: rgb(50, 120, 240);
+          background-color: rgb(50, 133, 190);
           border: 1px solid rgb(42, 112, 232);
         }
+        .night-mode .tie-button-blue {
+          background-color: rgb(70, 90, 110);
+          color: #ffffff;
+        }
+        .night-mode .tie-button-blue:hover {
+          background-color: rgb(85, 115, 150);
+        }
+        .tie-button-green {
+          background-color: rgb(110, 150, 90);
+          color: #ffffff;
+        }
+        .tie-button-green:hover {
+          background-color: rgb(50, 133, 90);
+          border: 1px solid rgb(42, 112, 132);
+        }
+        .night-mode .tie-button-green {
+          background-color: rgb(70, 90, 10);
+          color: #ffffff;
+        }
+        .night-mode .tie-button-green:hover {
+          background-color: rgb(85, 115, 50);
+        }
```

Comment:
Any reason why above does not follow the same convention as night-mode rules below? If there is a reason for the different convention, what about alphabetical order?

Reply:
This is also deliberate, but does introduce an inconsistency. I want to avoid having a whole bunch of "night-mode" classes scattered throughout the HTML, and instead depend on a top-level night-mode class; hence this way of doing it. (I'm happy to refactor the other usages in line with this too.)

I also wanted to keep the night-mode classes near the main classes, because the two are strongly related, and I want there to be a visual reminder that, when someone edits the main class, they should also at least take a look at the night-mode class. So I propose leaving the order as-is.

So, in response to this comment, I plan to refactor the stuff mentioned in the first paragraph but leave the code as-is in terms of ordering. Does that sound OK?

Reply:
Consistency is good (would be great if existing rules are updated to follow new convention).

If we are no longer following strict alphabetical convention, would be good to note that and how one should determine the order (it feels a bit tricky to say it's alphabetical order unless the author or reviewer feels a different order is better based on relationships). I think there is already the potential for other exceptions, such as where cascading rules may be impacted by order; so, adding another exception based on relationships might make any ordering rule more difficult to follow IMO.

Reply:
Per offline conversation, decided that, although there's a rule for alphabetizing attributes within a CSS rule, there is no guideline for alphabetizing rules -- so, leaving current order as-is.

Reply:
Also, I've updated the other night-mode rules to follow the new convention (and reduce duplication).